aiidalab_widgets_base/structures.py

```diff
@@ -215,6 +215,14 @@ def _sync_structure_node(self):
         else:
             structure_node = self._convert_to_structure_node(self.input_structure)
         self.set_trait('structure_node', structure_node)
+        
+    def _validate_and_fix_ase_cell(self, ase_structure, vacuum_ang=10.0):
+        cell = ase_structure.cell
+        no_cell = cell[0][0] < 0.1 or cell[1][1] < 0.1 or cell[2][2] < 0.1
+        if no_cell:
```

Comment:
maybe merge those two lines.

Reply:
Also, strictly speaking, this check is not correct. One should check the vector length, not just one its component.

Reply:
If any of the diagonal components is 0, I would say the cell is faulty

Reply:
no, it is not correct, comments @cpignedoli?